.github/workflows/deploy.yml

```diff
@@ -528,20 +533,28 @@ jobs:
     if: ${{ github.ref == 'refs/heads/production' && fromJSON(needs.checkDeploymentSuccess.outputs.IS_AT_LEAST_ONE_PLATFORM_DEPLOYED) }}
     needs: [checkDeploymentSuccess]
     steps:
-      - uses: actions/checkout@v4
-        name: Checkout
+      - name: Checkout production branch
+        uses: actions/checkout@v4
 
       - name: Get current app version
         run: echo "PRODUCTION_VERSION=$(npm run print-version --silent)" >> "$GITHUB_ENV"
 
+      - name: Download all workflow run artifacts
+        uses: actions/download-artifact@v4
+
+      - name: Rename web and desktop sourcemaps artifacts before assets upload in order to have unique ReleaseAsset.name
```

Comment:
Why is the rename necessary after download? Shouldn't it already be called `./desktop-sourcemaps/desktop-merged-source-map.js.map` at this point?